components/server/src/orgs/organization-service.ts

```diff
@@ -187,16 +187,16 @@ export class OrganizationService {
         if (userId) {
             await this.auth.checkPermissionOnOrganization(userId, "write_members", orgId);
         }
-        if (role !== "owner") {
```

Comment:
If this is called a second time on the first user of an org, we would try to remove the `owner` role which would fail the hole operation. We instead now keep the first and only owner as an owner.

Reply:
Trying to understand the comment.
* This if-statement was checking the membership downgrade to guarantee there is at least a single owner left if the downgrade would be executed. ✔️ 
* Why would it be "called a second time on the first user of an org" ❓

Reply:
> Why would it be "called a second time on the first user of an org"

That is because of the extra call in iam-session-app we added two days ago.